proto/claim/mission.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+package tendermint.spn.claim;
+
+option go_package = "github.com/tendermint/spn/x/claim/types";
+
+import "gogoproto/gogo.proto";
+
+message Mission {
+  uint64 ID = 1;
```

Comment:
```suggestion
  uint64 missionID = 1;
```

Reply:
This is what we have for other types. It can help depending on the variable name

Otherwise it should be lowercase `id`